terracumber-cli

```diff
@@ -108,6 +113,24 @@ def read_config(tf_file):
     return config
 
 
+def open_bastion_channel(args, tf_vars, bst_creds, ctl_creds):
+    """Update bastion and controller update"""
+    ctl = get_controller_hostname(args, tf_vars)
+    ctl_creds['hostname'] = ctl
```

Comment:
As car as I can tell, you are not using `ctl` later, so you should be able to do:
```suggestion
    ctl_creds['hostname'] = get_controller_hostname(args, tf_vars)
```